src/components/header/header_alert/header_alert.tsx

```diff
@@ -22,36 +26,27 @@ export const EuiHeaderAlert: FunctionComponent<EuiHeaderAlertProps> = ({
   date,
   text,
   title,
+  badge,
   ...rest
 }) => {
   const classes = classNames('euiHeaderAlert', className);
 
   return (
-    <EuiI18n token="euiHeaderAlert.dismiss" default="Dismiss">
-      {(dismiss: string) => (
-        <div className={classes} {...rest}>
-          <EuiButtonIcon
-            aria-label={dismiss}
-            iconType="cross"
-            size="s"
-            className="euiHeaderAlert__dismiss"
-          />
-
-          <div className="euiHeaderAlert__title">{title}</div>
-
-          <div className="euiHeaderAlert__text">{text}</div>
-
-          <EuiFlexGroup justifyContent="spaceBetween">
-            <EuiFlexItem grow={false}>
-              <div className="euiHeaderAlert__action euiLink">{action}</div>
-            </EuiFlexItem>
-
-            <EuiFlexItem grow={false}>
-              <div className="euiHeaderAlert__date">{date}</div>
-            </EuiFlexItem>
-          </EuiFlexGroup>
-        </div>
+    <div className={classes} {...rest}>
```

Comment:
I _think_ this could be better represented as an [`<article>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article)... What do other people think? I'm not 100% certain.

If we do go the `<article>` route, we can then also wire up the heading like so:
```suggestion
    <article aria-labelledby={headingId} className={classes} {...rest}>
```
(And then we'd add a `headingId` to the heading element below.

Reply:
I agree, using `article` here seems semantically correct.

Reply:
I replaced `div` with `article` and added `aria-labelledby`.